README.md

```diff
@@ -89,35 +109,49 @@ import { validate } from 'ember-validity-modifier';
 
 export default class MyForm extends Component {
   @action
-  async validateAndSubmit(event) {
-    let { target: form } = event;
-    event.preventDefault();
-    await validate(...form.elements);
-    if (form.reportValidity()) {
-      console.log('Fake submit action', Object.fromEntries(new FormData(form)));
-    }
+  validateAndSubmit({ target: form }) {
+    console.log('Fake submit action', Object.fromEntries(new FormData(form)));
   }
 }
 ```
 
-#### Example with `validator-update` event
+#### Example with `validateDidInsert` argument
 
-To validate the form state on initial render and any time its dependent arguments change, add the `'validator-update'` event to the list of events passed in via the `on` argument.
+To validate the form state on initial render add `validateDidInsert=true`.
 
 ```hbs
-  <input {{validity (fn this.matchTo this.match) on="change,validator-update"}}>
+  <input {{validity
+    (fn this.matchTo this.match)
+    on="change"
+    validateDidInsert=true
```

Comment:
Maybe eager?

Reply:
We tried eager in the past. Every time it was used I noticed myself and others confused by it. I think the term is to broad to be useful in this context.